src/openrct2/interface/Theme.cpp

```diff
@@ -165,6 +165,7 @@ WindowThemeDesc WindowThemeDescriptors[] =
     { THEME_WC(WC_EDITOR_SCENARIO_BOTTOM_TOOLBAR), STR_THEMES_WINDOW_BOTTOM_TOOLBAR_SCENARIO_EDITOR,       COLOURS_3(TRANSLUCENT(COLOUR_LIGHT_BROWN), TRANSLUCENT(COLOUR_LIGHT_BROWN), TRANSLUCENT(COLOUR_MOSS_GREEN)                     ) },
     { THEME_WC(WC_TITLE_EDITOR),                   STR_TITLE_EDITOR_TITLE,       COLOURS_3(COLOUR_GREY,                     COLOUR_OLIVE_GREEN,              COLOUR_OLIVE_GREEN                                 ) },
     { THEME_WC(WC_TILE_INSPECTOR),                 STR_TILE_INSPECTOR_TITLE,       COLOURS_2(COLOUR_LIGHT_BLUE,               COLOUR_LIGHT_BLUE                                                                   ) },
+    { THEME_WC(WC_VIEW_CLIPPING),                 STR_TILE_INSPECTOR_TITLE,       COLOURS_1(COLOUR_LIGHT_BLUE                ) },
```

Comment:
Missing one space before the `STR_` constant. And still should be another colour. What do you think of the light brown used by the map view and additional viewports? Those seem to be the same 'group' or views.